homeassistant/components/renson/sensor.py

```diff
@@ -0,0 +1,310 @@
+        field=DAYTIME_FIELD,
+        raw_format=False,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="NIGHTTIME_FIELD",
+        name="Start night time",
+        field=NIGHTTIME_FIELD,
+        raw_format=False,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="DAY_POLLUTION_FIELD",
+        name="Day pollution level",
+        field=DAY_POLLUTION_FIELD,
+        raw_format=False,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="NIGHT_POLLUTION_FIELD",
+        name="Night pollution level",
+        field=NIGHT_POLLUTION_FIELD,
+        raw_format=False,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="CO2_THRESHOLD_FIELD",
+        name="CO2 threshold",
+        field=CO2_THRESHOLD_FIELD,
+        raw_format=False,
+        native_unit_of_measurement="ppm",
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="CO2_HYSTERESIS_FIELD",
+        name="CO2 hysteresis",
+        field=CO2_HYSTERESIS_FIELD,
+        raw_format=False,
+        native_unit_of_measurement="ppm",
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="BYPASS_TEMPERATURE_FIELD",
+        name="Bypass activation temperature",
+        field=BYPASS_TEMPERATURE_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    RensonSensorEntityDescription(
+        key="BYPASS_LEVEL_FIELD",
+        name="Bypass level",
+        field=BYPASS_LEVEL_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="%",
+    ),
+    RensonSensorEntityDescription(
+        key="FILTER_PRESET_FIELD",
+        name="Filter preset time",
+        field=FILTER_PRESET_FIELD,
+        raw_format=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="days",
+    ),
+)
+
+
+class RensonSensor(RensonEntity, SensorEntity):
+    """Get a sensor data from the Renson API and store it in the state of the class."""
+
+    def __init__(
+        self,
+        description: RensonSensorEntityDescription,
+        api: RensonVentilation,
+        coordinator: RensonCoordinator,
+    ) -> None:
+        """Initialize class."""
+        super().__init__(description.key, api, coordinator)
+
+        self.field = description.field
+        self.entity_description = description
+
+        self._state = None
+        self.data_type = description.field.field_type
+        self.raw_format = description.raw_format
+
+    @property
+    def native_value(self):
```

Comment:
Either we remove this property or we move the code in `_handle_coordinator_update` here and just return the value and remove that callback.